### DISCUSSION TITLE
File upload limit.

### DISCUSSION THREAD

**Discussion by microm3nn** [pimcore/pimcore#15257] [Q&A / Support]
Hi 
Can anyone explain how to increase file upload limit? I use this -> [Pimcore Demo](https://github.com/pimcore/demo#follow-these-steps) to install Pimcore demo. Now limit is set to 100MB. In the official documentation there is nothing about increasing upload limit.

**Comment by mattamon** [comment]
https://pimcore.com/docs/platform/Pimcore/Installation_and_Upgrade/System_Requirements/ -> the file upload limit is defined by the server settings and heavily depends  on your server setup. 
For apache e.g. https://www.rumahtulip.nl/blog/index.php/apache/how-to-increase-upload-file-size-limit-on-php-apache
https://stackoverflow.com/questions/26717013/how-to-edit-nginx-conf-to-increase-file-size-upload
